admin_pages/messages/templates/ee_msg_m_settings_content.template.php

```diff
@@ -23,9 +23,9 @@
             <?php echo $nonce; // already escaped ?>
         </span>
         <div class="switch">
-            <?php $checked = $on_off_status ? ' checked="checked"' : ''; ?>
+            <?php $checked = $on_off_status ? 'checked="checked"' : ''; ?>
             <input id="ee-on-off-toggle-<?php echo esc_attr($messenger); ?>" type="checkbox"
-                   class="ee-on-off-toggle ee-toggle-round-flat"<?php echo $checked; ?>
+                   class="ee-on-off-toggle ee-toggle-round-flat" <?php echo $checked; ?>
```

Comment:
as much as I hate to suggest it... 😞 

```suggestion
                   class="ee-on-off-toggle ee-toggle-round-flat" <?php echo esc_attr($checked); ?>
```

Reply:
If you recall, recently we had a problem with using `esc_attr` function for the attributes that have double quotes. To avoid that issue, I changed `checked="checked"` to `checked` and then added `esc_attr` function. `checked`, `selected` and `disabled` are enough and we don't need the second part.